### PR TITLE
Add a way to set a Vagrant boot_timeout value

### PR DESCRIPTION
**Pull request by jonmccreery**

### CODE REVIEW

File: lib/rzo/app/templates/Vagrantfile.erb [hunk 1]
```diff
@@ -16,6 +16,9 @@ Vagrant.configure(2) do |config|
     cfg.vm.box_url = <%= nc['box_url'].inspect %>
     cfg.vm.box_download_checksum = <%= nc['box_download_checksum'].inspect %>
     cfg.vm.box_download_checksum_type = <%= nc['box_download_checksum_type'].inspect %>
+    <%- if nc['boot_timeout'] -%>
```
Comment (ghoneycutt) [inline lib/rzo/app/templates/Vagrantfile.erb:19]: Awesome! Thank you!

Could you please add documentation under https://github.com/ghoneycutt/rizzo#nodes and mention that Vagrant's default is 300 seconds and that the value should be denoted in seconds.

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

**Comment by ghoneycutt** [comment]
@jeffmccune Could you please add testing for this

**Comment by jeffmccune** [comment]
This is great @jonmccreery I added some testing an a note in the README in #28 .  I based the branch on your pull request.

**Comment by jonmccreery** [comment]
Whoops.. Sorry @jeffmccune.  I just noticed that you also added line of doc in #28  along with that test, which is going to run into 8ff3775.

**Comment by jeffmccune** [comment]
> Whoops.. Sorry @jeffmccune. I just noticed that you also added line of doc in #28 along with that test, which is going to run into 8ff3775.

No problem, I'll rebase.

**Comment by ghoneycutt** [comment]
Tests added in PR #28

**Pull request closed without merging**